### PR TITLE
Finished issue #6

### PR DESCRIPTION
**Pull request by zhengye1**
Issue #6 done, also I change the all other route to the path /404 and make the behavior is same. 

`ng test` now is passed and also for `ng e2e`, but I don't think I have a good test coverage... If someone else who wants to help can try to add more test on it.

### CODE REVIEW

File: frontend/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -1,32 +1,76 @@
...
-  it(`should have as title 'app works!'`, async(() => {
-    const fixture = TestBed.createComponent(AppComponent);
-    const app = fixture.debugElement.componentInstance;
-    expect(app.title).toEqual('app works!');
-  }));
+  // it(`should have as title 'app works!'`, async(() => {
+  //   let fixture = TestBed.createComponent(AppComponent);
+  //   const app = fixture.debugElement.componentInstance;
+  //   expect(app.title).toEqual('app works!');
+  // }));
```
Comment (bfwg) [inline frontend/src/app/app.component.spec.ts]: Is there a reason we keep those comment?

**Comment by bfwg** [comment]
Nice work, LGTM 👍

File: frontend/src/app/app.component.spec.ts (hunk 1 above)
Comment (zhengye1) [inline frontend/src/app/app.component.spec.ts]: Actually we can delete it. I forgot to clean that up..

Comment (bfwg) [inline frontend/src/app/app.component.spec.ts]: No worries, let's clean it in later iterations .

**Pull request merged**